cypress/e2e/17-stickies.cy.ts

```diff
@@ -0,0 +1,289 @@
+import { WorkflowPage as WorkflowPageClass } from '../pages/workflow';
+
+const workflowPage = new WorkflowPageClass();
+
+describe('Canvas Actions', () => {
+	beforeEach(() => {
+		cy.resetAll();
+		cy.skipSetup();
+		workflowPage.actions.visit();
+		cy.waitForLoad();
+	});
+
+	it('adds sticky to canvas with default text and position', () => {
+		workflowPage.getters.addStickyButton().should('not.be.visible');
+		workflowPage.actions.addSticky();
+
+		workflowPage.getters.stickies().should('have.length', 1)
+			.and(($el) => {
+				expect($el).to.have.css('top', '340px');
+				expect($el).to.have.css('left', '400px');
+				expect($el).to.have.css('height', '160px');
+				expect($el).to.have.css('width', '240px');
```

Comment:
Default position and default dimensions are often repeated so we could generalize.

Reply:
fair.. will try to see if I could standarize this..